File: VB/SchedulerReportPreviewTest/Root.Master.vb
Imports System
Imports DevExpress.Web

Namespace SchedulerReportPreviewTest

    Public Partial Class Root
        Inherits Web.UI.MasterPage

        Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
            MainSplitter.GetPaneByName("HeaderPane").Size = If(Equals(ASPxWebControl.GlobalTheme, "Moderno"), 101, 87)
            MainSplitter.GetPaneByName("HeaderPane").MinSize = If(Equals(ASPxWebControl.GlobalTheme, "Moderno"), 101, 87)
        End Sub
    End Class
End Namespace

File: CS/SchedulerReportPreviewTest/Root.Master.cs
using System;
using DevExpress.Web;

namespace SchedulerReportPreviewTest {
    public partial class Root : System.Web.UI.MasterPage {
        protected void Page_Load(object sender, EventArgs e) {
            MainSplitter.GetPaneByName("HeaderPane").Size = ASPxWebControl.GlobalTheme == "Moderno" ? 101 : 87;
            MainSplitter.GetPaneByName("HeaderPane").MinSize = ASPxWebControl.GlobalTheme == "Moderno" ? 101 : 87;
        }
    }
}